run_dir/static/scss/custom.scss

```diff
@@ -0,0 +1,7 @@
+// Custom.scss
+// Option A: Include all of Bootstrap
+
+
+$font-size-root: .8125rem; // 13px instead of default 16px
+
+@import "/Users/johannes.alneberg/node_modules/bootstrap/scss/bootstrap";
```

Comment:
Umm, would this still work on other systems?

Reply:
🤦‍♂️